packages/documentation-site/patternfly-docs/content/developer-resources/dark-theme-handbook.md

```diff
@@ -0,0 +1,65 @@
+---
+id: Dark theme handbook
+title: PatternFly dark theme handbook
+section: developer-resources
+---
+
+## Enabling dark theme
+
+Dark theme styles ship with PatternFly CSS by default. To enable dark theme, add the class  `pf-v5-theme-dark`  to the `<html>` tag of your application. This class can be added dynamically to toggle dark theme on and off in your application.
+
+Dark theme can also be applied based on the browser’s `prefers-color-scheme` media query. The recommended way to apply dark theme based on this media query is to use Javascript(JS) to detect the media query and apply the `pf-theme-dark` class via the JS code. This allows a product to simply manipulate a single class to turn dark theme on or off based on user preference. Adam Argyle wrote [a great article](https://web.dev/building-a-theme-switch-component/#javascript) on using JS to manage dark theme preference, using the `prefers-color-scheme` media query alongside a manual button to toggle dark theme on or off. 
+
+## What’s changed?
+The following are some notable changes and differences between light and dark themes. 
+
+| Color palette changes | Variable(s) affected | New or updated color values|
```

Comment:
This table looks a little odd with so much white space for the empty columns. I wonder if it would be too repetitive to have something like

#### Black color palette
Black color palette updated and a new lighter gray was added

| Variable(s) affected | New or updated color values|
| -------------------- | -------------------- |
| --pf-v5-global--palette--black-50  | #e0e0e0 |
| --pf-v5-global--palette--black-100 | #c6c7c8 |
| --pf-v5-global--palette--black-200 | #aaabac |
| --pf-v5-global--palette--black-300 | #868789 |
| --pf-v5-global--palette--black-400 | #57585a |
| --pf-v5-global--palette--black-500 | #444548 |
| --pf-v5-global--palette--black-600 | #36373a |
| --pf-v5-global--palette--black-700 | #26292d |
| --pf-v5-global--palette--black-800 | #1b1d21 |
| --pf-v5-global--palette--black-850 | #212427 |
| --pf-v5-global--palette--black-900 | #0f1214 |
| --pf-v5-global--palette--black-1000| #030303 |

#### Red color palette
Two new red colors added 

| Variable(s) affected | New or updated color values|
| -------------------- | -------------------- |
| --pf-v5-global--palette--red-9999  | #fe5142 |
| --pf-v5-global--palette--red-8888  | #ff7468 |


#### Blue color palette
`blue-300` was updated to support an additional variation of primary blue'

| Variable(s) affected | New or updated color values|
| -------------------- | -------------------- |
| --pf-v5-global--palette--blue-300 | #1fa7f8 |

Reply:
I do like that better